CompuCell3D/CMakeLists.txt

```diff
@@ -28,8 +28,10 @@ IF(OPENMP_FOUND)
 message ("openmp c flags " "${OpenMP_C_FLAGS}")
 message ("openmp cxx flags " "${OpenMP_CXX_FLAGS}")
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS} ")
-#SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS} -std=c++11 -stdlib=libc++")
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+IF(APPLE)
```

Comment:
I would go even further and try to fine tune behavior depending if we are using clang compiler on OSX.  so
`if (APPLE and CMAKE_CXX_COMPILER_ID STREQUAL "Clang")`